src/rules/operator-no-unspaced/index.js

```diff
@@ -159,7 +159,7 @@ export function calculationOperatorSpaceChecker({ root, result, checker }) {
 
       // Media queries
       if (item.name === "media" || item.name === "import") {
-        mediaQueryParser(item.params).walk(node => {
+        mediaQueryParser(item.params).walk("", (node) => {
```

Comment:
@dryoma I tried installing `postcss-media-query-parser` 0.2.2 to remove this workaround, but it seems that you forgot to rebuild before releasing it to npm, and only the `src` folder has the correct changes.